File: BNOI - Group C/Balls/Balls.js
function solve(args) {
    var input = args[0].split(' ');
    var k = parseInt(input[1]);
    var boxes = [];
    var distinct = [];
    for (var i = 0; i < k; i++) {
        var element = parseInt(args[1 + i]);
        boxes.push(element);
        var found = true;
        for (var j = 0; j < distinct.length; j++) {
            if (distinct[j] === element) {
                found = false;
                break;
            }
        }
        if (found) distinct.push(element);
    }

    return parseInt(input[0]) - distinct.length;
}

File: Bulgarian Competitions/BNOI - Group C/Balls/Balls.cs
using System;
using System.Collections.Generic;
using System.Linq;

class Balls
{
    static void Main()
    {
        string[] input = Console.ReadLine().Split(' ');
        List<long> boxes = new List<long>();

        int k = int.Parse(input[1]);
        for (int i = 0; i < k; i++) boxes.Add(long.Parse(Console.ReadLine()));
        Console.WriteLine(long.Parse(input[0]) - boxes.Distinct().Count());
    }
}